### DISCUSSION TITLE
求问一下

### DISCUSSION THREAD

**Discussion by Tragedysaver1024** [share121/inter-knot#110] [其他]
![phone2 - 2024-08-09T181351 266](https://github.com/user-attachments/assets/a6884ca1-03a4-4332-bd08-fa325035d3b6)
今天熬夜蹲大厕的时候手机放在一边，录屏没关，然后我出厕所之后从录屏里面看到这个……
有没有好心人来解释一下这六个图标是什么意思？尤其是左边那几个图标似乎出现了不止一次

**Comment by Luli-Chen** [comment]
小眼睛好像是开了护眼模式会有

**Comment by Mu-7593** [comment]
高能耗应用和眼睛般的图标，莫非楼主的手机产生了自我意识？保险起见，建议楼主把手机送去做一下禁果测试

**Reply by Tragedysaver1024** [reply]
我一个在不懂叫什么基金会工作的朋友说还好我福大命大没有抬头……不懂他是在装神弄鬼还是作甚

**Comment by Luli-Chen** [comment]
小心侵蚀之律者（

**Comment by Tragedysaver1024** [comment]
回1，2，3楼（不懂为什么我的回复显示不出来）
我一个在不懂叫什么基金会工作的朋友说得亏我昨晚福大命大没有抬头，天知道他是在装神弄鬼还是作甚

**Comment by Tragedysaver1024** [comment]
他好像不太愿意再跟我透露太多了，只是跟我说凡是遇到这种情况的人似乎都寄了

**Comment by Tragedysaver1024** [comment]
我再多问他他就只贴了这张图片给我，谁告诉我图片里面的“Keter”是啥意思来的

[![pASEMGD.jpg](https://s21.ax1x.com/2024/08/09/pASEMGD.jpg)](https://imgse.com/i/pASEMGD)

**Comment by Tragedysaver1024** [comment]
还有人吗，怎么不说话啊，你们不说话我害怕……

**Comment by AL-Pinecore** [comment]
emm... 这个委托我接不了

**Comment by wushanef** [comment]
难道是ghost

**Comment by Luli-Chen** [comment]
啊，他说的是SCP基金会吧，“Keter”是指代难以收容的收容物

**Comment by Luli-Chen** [comment]
帖主注意安全，最好不要过多了解这个基金会相关，知道太多会有人员上门来清除你的记忆

**Comment by Tragedysaver1024** [comment]
谁还记得我发帖问这几个图标是什么意思的

**Comment by Luli-Chen** [comment]
乐的，歪楼了

**Comment by Tragedysaver1024** [comment]
> 帖主注意安全，最好不要过多了解这个基金会相关，知道太多会有人员上门来清除你的记忆

话说在新艾利都这个异常人人皆知的地方这个基金会用得着隐瞒吗

**Comment by Luli-Chen** [comment]
> > 帖主注意安全，最好不要过多了解这个基金会相关，知道太多会有人员上门来清除你的记忆
> 
> 话说在新艾利都这个异常人人皆知的地方这个基金会用得着隐瞒吗

不知道诶，但是基金会涉及的异常很深，不该被我们知道的还是会遮掩的吧

**Comment by Tragedysaver1024** [comment]
我那朋友告诉我这玩意在数据库的代号是“不要向上看”……
这下一清二楚了

**Comment by Luli-Chen** [comment]
哥们以后上厕所只能低头玩手机了（悲
